shared/actions/profile/__test__/profile-path.test.js

```diff
@@ -0,0 +1,170 @@
+// @noflow
+/* eslint-env jest */
+import {getProfilePath} from '../../../constants/profile'
+import * as I from 'immutable'
+import type {PropsPath} from '../../../route-tree/index'
+import {peopleTab} from '../../../constants/tabs'
+
+// These tests are for navigation with newPeopleTab: false
+// when the new people tab is going to be released these
+// expectedPaths must change to reflect the new root behavior
```

Comment:
This is just adding an extra `peopleTab` to the root of some of the expected results